packages/utils/messages.ts

```diff
@@ -34,6 +53,8 @@ const WASM_TYPES: WasmMsgType[] = [
   'clear_admin',
 ]
 
+export type CosmWasmStargateMsg = { stargate: { type_url: string; value: any } }
```

Comment:
can you move this to `@dao-dao/types/contracts/common` where `CosmosMsgFor_Empty` is defined? in fact, I think that type already contains a `stargate` type. Any reason why we have to redefine this?